rclcpp_components/CMakeLists.txt

```diff
@@ -41,6 +41,15 @@ ament_target_dependencies(component_container
   "rclcpp"
 )
 
+set(install_dir ${PROJECT_BINARY_DIR})
```

Comment:
I would make this name more precise, e.g. `node_main_template_install_dir`. Also, I think it should work if we set it to something other than the build directory, e.g. `lib` or `share`.